admin/bin/parse-migrations-csv.js

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env node
+'use strict'
+
+require('dotenv').config()
+const csv = require('fast-csv')
+const fs = require('fs')
+const poolService = require('../services/data-access/sql.pool.service')
```

Comment:
This isn't needed anymore.

Reply:
@danwilliams i would suggest a refresh from master, because that file no longer exists (sql.pool.service)